styles.css

```diff
@@ -0,0 +1,116 @@
+/* stylelint-disable max-empty-lines */
+
+
+/* stylelint-disable no-missing-end-of-source-newline */
+
+
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+
+
+/* stylelint-disable indentation */
+
```

Comment:
- [ ] Please note, you should not disable any stylelint constraints since they are rules to make your code clean. Kindly remove them and fix the errors automatically by using this command `npx stylelint "**/*.{css,scss}" --fix`